src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -167,7 +163,11 @@ public String getApiUri() {
             for (GHRepository repo : user.listRepositories()) {
                 add(listener, observer, repo);
             }
+            return;
         }
+
+        listener.getLogger().format("%n%s does not correspond to a known GitHub User Account or Organization%n%n", repoOwner);
+        throw new InterruptedException();
```

Comment:
@jglick Then, we need a new `Exception` or handling in a different way. As I mentioned [here](https://issues.jenkins-ci.org/browse/JENKINS-33815?focusedCommentId=251922&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-251922) I don't want to show traces to the user.